Dockerfile

```diff
@@ -1,16 +1,24 @@
 FROM ubuntu:focal-20210827
 
+ARG USER=haskell
+ARG UID=1000
+ARG GID=1000
+
 ARG DEBIAN_FRONTEND=noninteractive
 ARG STACK_VERSION=2.7.3
 
 ENV LANG=C.UTF-8
 
-# Make sure this path includes initdb for the usage of tmp-postgres
-ENV PATH=/usr/lib/postgresql/13/bin/:$PATH
+ENV PATH=/home/$USER/.ghcup/bin:/stack/bin:$PATH
+
+# Create a default home for the default user & allow any user to sudo
+RUN useradd --create-home --uid "$UID" "$USER" \
```

Comment:
I know that the group will be created automatically by `useradd`, but do we need to somehow specify the `$GID`?